packages/forms/src/UIForm/utils/labels.js

```diff
@@ -9,8 +9,9 @@ export const getLabelProps = (title, labelProps, hint) => {
 	}
 	return {
 		children: (
-			<StackHorizontal gap="XS" align="center">
+			<StackHorizontal gap="XXS" align="center">
 				{title}
+				{required && '*'}
```

Comment:
I prefer if you can use a classic css approach.
Here you can't for example make it red

```
.required label:after {
    color: #e32;
    content: ' *';
    display:inline;
}
```

Reply:
Ok good point 
I could also change it on design system side , the idea was to have the same behavior